modules/bmc_check/esbmccheck.py

```diff
@@ -712,7 +716,7 @@ def hassuccessfulfromesbmc(_esbmcoutput):
 
 
     def checkwithkparallel(self, _cprogrampath, _actual_ce, _listtmpfiles):
-        result_parallel = commands.getoutput(self.esbmcpath + " " + self.esbmc_arch + " " +
```

Comment:
Did we remove the architecture information?

Reply:
Yes, the esbmc architecture was removed from the entire code .